### ISSUE TITLE
Expose "Set Brightness"

### ISSUE THREAD

**Issue by robcos** [bigbadblunt/homeassistant-lightwave2#26]
It would be nice to be able to set the brightness of a light without turning it on.
This can be useful in automations like "at 10pm, set bathroom light brightness to 20%" so that when I go for a pee I don't get blinded in the middle of the night.
I poked with the code and saw that this should be possible with lightwave, which is a feature not available otherwise with many other light integrations.

**Comment by bigbadblunt** [comment]
@robcos mixed news! I've added a service call lightwave2.set_brightness that does what you request. The bad news is this only works for Generation 2 lights. For Gen 1 lights, any change to the brightness level is interpreted as a request to turn on.

I don't know what kit you have, but fingers crossed it's a gen 2 light you're using.

Let me know if this works for you. I've pushed a version 3.1.0beta. I don't know which version you were using before, if it was 2.x then you'll need to delete and re-add the integration.
